src/lsdb/catalog/catalog.py

```diff
@@ -80,7 +85,7 @@ def head(self, n: int = 5) -> pd.DataFrame:
                 dfs.append(partition_head)
                 remaining_rows -= len(partition_head)
         if len(dfs) > 0:
-            return pd.concat(dfs)
+            return npd.NestedFrame(pd.concat(dfs))
```

Comment:
Sometimes, it's `npd.NestedFrame`, sometimes it's just `NestedFrame`. I get that dask also overloads the `DataFrame` name for both pandas and distributed data, but it makes the code that uses both a little weird to read. There are a few possible conventions we could use to make it clearer, but we should discuss.

Reply:
Yeah, I agree it's confusing. Maybe we can discuss at the weekly tomorrow?

Reply:
As discussed, we now use `nd.NestedFrame` and `npd.NestedFrame`